src/prefect/environments/execution/remote/environment.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any
+
+import cloudpickle
+
+from prefect.environments.execution import Environment
+from prefect.environments.storage import Storage
+from prefect.utilities import logging
+from prefect.utilities.configuration import set_temporary_config
+
+
+class RemoteEnvironment(Environment):
+    """
+    RemoteEnvironment is an environment which takes in information about an executor
+    and runs the flow in place using that executor.
+
+    Example:
+    ```python
+    # using a RemoteEnvironment w/ an existing Dask cluster
+
+    env = RemoteEnvironment(
+        executor="prefect.engine.executors.DaskExecutor",
+        executor_kwargs={"address": "tcp://dask_scheduler_address"}
+    )
+
+    f = Flow("dummy flow", environment=env)
+    ```
+
+    Args:
+        - executor (str, optional): an importable string to an executor class; defaults
+            to `prefect.engine.executors.SynchronousExecutor`
+        - executor_kwargs (dict, optional): a dictionary of kwargs to be passed to
+            the executor; defaults to an empty dictionary
+    """
+
+    def __init__(
+        self,
+        executor: str = "prefect.engine.executors.SynchronousExecutor",
```

Comment:
Just out of curiosity, would it be better to use `prefect.config.engine.executor.default_class` here?  That way it represents user defaults, and can change in the future without us having to remember it's existence here